src/main/java/org/fundacionjala/coding/nestor/Customer.java

```diff
@@ -0,0 +1,77 @@
+package org.fundacionjala.coding.nestor;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+public class Customer {
+    private String _name;
+    private Vector<Rental> _rentals = new Vector<Rental>();
+
+    public Customer(String name) {
+        _name = name;
+    }
+
+    public void addRental(Rental arg) {
+        _rentals.addElement(arg);
+    }
+
+    public String getName() {
+        return _name;
+    }
+
+	public double getTotalAmount() {
+        double totalAmount = 0;
+        Enumeration<Rental> rentals = _rentals.elements();
+
+        while (rentals.hasMoreElements()) {
+            Rental element = (Rental) rentals.nextElement();
+            totalAmount += element.getCharge();
+        }
+
+        return totalAmount;
+    }
+    
+    public String statement() {
+        double totalAmount = 0;
+        int frequentRenterPoints = 0;
+        Enumeration<Rental> rentals = _rentals.elements();
+        String result = "Rental Record for " + getName() + "\n";
+        while (rentals.hasMoreElements()) {
+            double thisAmount = 0;
+            Rental each = (Rental) rentals.nextElement();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "each". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unnecessary cast to "Rental". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1905)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "each" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)